quickwit/quickwit-doc-mapper/src/default_doc_mapper/field_mapping_entry.rs

```diff
@@ -353,6 +353,15 @@ impl TextIndexingOptions {
         }
     }
 
+    fn from_parts_concatenate(
+        tokenizer: Option<QuickwitTextTokenizer>,
+        record: Option<IndexRecordOption>,
+    ) -> anyhow::Result<Self> {
+        let text_index_options_opt = Self::from_parts_text(true, tokenizer, record, false)?;
+        let text_index_options = text_index_options_opt.expect("concatenate field must be indexed");
```

Comment:
Can this be triggered by user input?

Reply:
`Self::from_parts_text` the line above will always return Ok(Some(_)) if it's first parameter is `true`